submitted_models/robotika_freyja_sensor_config_1/launch/spawner.rb

```diff
@@ -0,0 +1,76 @@
+def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
+  <<-HEREDOC
+  <spawn name='#{_name}'>
+    <name>#{_name}</name>
+    <allow_renaming>false</allow_renaming>
+    <pose>#{_x} #{_y} #{_z + 0.15} #{_roll} #{_pitch} #{_yaw}</pose>
+    <world>#{_worldName}</world>
+    <is_performer>true</is_performer>
+    <sdf version='1.6'>
+      <include>
+        <name>#{_name}</name>
+        <uri>#{_modelURI}</uri>
+        <!-- Diff drive -->
+        <plugin filename=\"libignition-gazebo-diff-drive-system.so\"
+                name=\"ignition::gazebo::systems::DiffDrive\">
+          <left_joint>front_left_wheel_joint</left_joint>
+          <left_joint>rear_left_wheel_joint</left_joint>
+          <right_joint>front_right_wheel_joint</right_joint>
+          <right_joint>rear_right_wheel_joint</right_joint>
+          <wheel_separation>0.38</wheel_separation>
+          <wheel_radius>0.1275</wheel_radius>
+          <topic>/model/#{_name}/cmd_vel_relay</topic>
+          <min_velocity>-2.5</min_velocity>
+          <max_velocity>2.5</max_velocity>
+          <min_acceleration>-10.6</min_acceleration>
+          <max_acceleration>10.6</max_acceleration>
+        </plugin>
+        <!-- Publish robot state information -->
+        <plugin filename=\"libignition-gazebo-pose-publisher-system.so\"
+          name=\"ignition::gazebo::systems::PosePublisher\">
+          <publish_link_pose>true</publish_link_pose>
+          <publish_sensor_pose>true</publish_sensor_pose>
+          <publish_collision_pose>false</publish_collision_pose>
+          <publish_visual_pose>false</publish_visual_pose>
+          <publish_nested_model_pose>#{$enableGroundTruth}</publish_nested_model_pose>
+          <use_pose_vector_msg>true</use_pose_vector_msg>
+          <static_publisher>true</static_publisher>
+          <static_update_frequency>1</static_update_frequency>
+        </plugin>
+        <!-- Battery plugin -->
+        <plugin filename=\"libignition-gazebo-linearbatteryplugin-system.so\"
+          name=\"ignition::gazebo::systems::LinearBatteryPlugin\">
+          <battery_name>linear_battery</battery_name>
+          <voltage>25.2</voltage>
+          <open_circuit_voltage_constant_coef>25.2</open_circuit_voltage_constant_coef>
+          <open_circuit_voltage_linear_coef>-7.2</open_circuit_voltage_linear_coef>
+          <initial_charge>5</initial_charge>
+          <capacity>5</capacity>
```

Comment:
@nkoenig Sorry, I do not understand your change of capacity from 20 Ah, which corresponds to the real robot, to 5 Ah, which I do not know where it comes from. Isn't it cutting the driving time to a quarter?

Reply:
Validation data was not included to support the specified battery life of 14400 seconds as required by the [Model Preparation Guide](https://subtchallenge.com/resources/Simulation_Model_Preparation_Guide.pdf). The battery life has been reduced to 60 minutes to match the other UGVs for now, but if the endurance test data is provided, we can adjust the model to match the robot's demonstrated battery life.